main.go

```diff
@@ -34,6 +34,9 @@ var (
 	versionFlag = flag.Bool("version", false, "Show version and exit.")
 	httpFlag    = flag.String("http", ":7878", "HTTP service address.")
 	skipBrowser = flag.Bool("skipbrowser", false, "Skip opening browser.")
+	outputFile  = flag.String("output", "", "output file - just output .gv (dot) file, no http service")
+	svgFlag     = flag.Bool("svg", false, "use with -output, also output .svg file")
```

Comment:
Could you rather define `format` flag which has some default and can be more easily extended with new formats without introducing another flag in the future.